packages/create-app/templates/default-app/tools/development/db/0001-backstage-scaffold.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+set -e
+
```

Comment:
We're using `/scripts` as the foldername for these kinda things

Reply:
I'm curious if all of this is actually required? I've been able to just have a `.sql` file in the `/docker-entrypoint-initdb.d` directory with just a line for each `CREATE DATABASE`. No need for a shell script or granting privileges. Assuming this is purely for local development.

Reply:
Yeah viewing this as a workaround until we settle #1598

Reply:
I'm still not sure why this needs to be a shell script though? The docker `postgres` image supports [initialization](https://github.com/docker-library/docs/blob/master/postgres/README.md#initialization-scripts) from an `sql` file containing just `SQL`. I could be missing something, but I think this file could literally be:

```sql
# packages/create-app/templates/default-app/tools/development/db/0001-backstage-scaffold.sql
CREATE DATABASE backstage_plugin_auth;
CREATE DATABASE backstage_plugin_catalog;
```

Reply:
Yeah that's nicer, let's switch to that